db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBConnectionString.java

```diff
@@ -263,8 +266,77 @@ private EventStoreDBClientSettings parseSearchParams(boolean first) throws Parse
                 this.settings.throwOnAppendFailure(value.equals("true"));
                 break;
             }
+            case "keepAliveTimeout": {
+                try {
+                    long parsedValue = parseLong(value);
```

Comment:
**SUGGESTION:** It's not a blocker, might be handled eventually in the separate PR, but it might be worth to create a helper method for parsing those values to reduce the code redundancy. The same comment applies to the builder code.

Reply:
Added it in the backlog: https://github.com/EventStore/EventStoreDB-Client-Java/issues/61